ebu_tt_live/documents/ebuttd.py

```diff
@@ -46,8 +52,23 @@ def create_from_raw_binding(cls, binding):
         instance._ebuttd_content = binding
         return instance
 
+    def _get_bds(self):
```

Comment:
It just happens to be created right underneath this box :)

It is BindingDOMSupport. A set of mappings, including namespace identifiers and default namespaces that influence how the bindings are turned into xml.minidom.Document instances and then serialized to XML. E.g: tts:lineHeight can be ensured by providing namespace_prefix_map